content/en/actions/app_builder/build.md

```diff
@@ -16,70 +16,131 @@ further_reading:
-1. Bits AI sometimes asks clarification questions while it responds to your prompt. If this happens, enter a response, then press **Enter**. 
-1. After Bits AI adds an action to the app, you are prompted to either **Mock with AI Data** or **skip configuration**. Configuring your action during UI agent execution helps connect the data types to other components.
-1. After Bits AI responds to your prompt, you can click **Edit with AI** or manually [customize your app][15].
+    - While it's responding, Bits AI asks you to set up a connection to build the app with real data. You can complete this step, or skip it to build the app layout faster without data.
+    - Bits AI sometimes asks clarification questions while it responds to your prompt. If this happens, enter a response, then press **Enter**. 
+    - After Bits AI adds an action to the app, you are prompted to either **Mock with AI Data** or **Skip**. Configuring your action during UI agent execution helps connect the data types to other components.
+1. After Bits AI responds to your prompt, the following steps are optional: 
+    - To change the app name and description, click the app name.
+    - Click **Edit with AI** or manually [customize your app][15].
+    - To preview the app, click **View**. Click **Edit** to return to the configuration view.
 1. To save the app, click **Save**.
-1. To preview the app, click **View**. Click **Edit** to return to the configuration view.
 1. When you're ready to publish your app, click **Publish**. Publishing an app makes it available to your dashboards.
 
 To iterate on an existing app:
-1. From an existing app, click **UI Agent Chat** (**<i class="icon-bits-ai"></i>**).
+1. From an existing app, click **Edit** then **Build with AI** (**<i class="icon-bits-ai"></i>**).
 1. Enter a detailed prompt for the behavior you'd like to add to your app. Include the integrations and actions you'd like to use.
 1. Press **Enter** to add the functionality to your app.
 
-### Create a custom app
+### Build an app from a blueprint
+
+Blueprints are helpful templates that cover common use cases. They come loaded with demo data that you can use to familiarize yourself with the app. Blueprints also showcase best practices for setting up app functionality and visual presentation.
+
+1. From [App Builder][1], click the [Blueprints][2] tab. Alternatively, click **New App**, then **Start with Blueprint** in the onboarding modal.
+1. Find the blueprint you want to use and click **Use Blueprint**.
+1. The following steps are optional: 
+    - Each blueprint template comes loaded with demo data. To customize the app, edit the **Connection** for each query.
+    - To add a [UI component](#app-canvas-and-components) to the app canvas, click **Add Component** (**<i class="icon-component-wui"></i>**) to open the **Components** tab. Click the component or drag it onto the canvas.
```

Comment:
I didn't initially see this step when I was looking at a blueprint (I chose the GitLab Manager one). I think this procedure is missing the "Open in Editor" click? Make sure you test your procedures end-to-end, especially with UI that might be subject to change, so you can prevent your users from getting lost.

And again, I have the same comment about thinking about what you're telling your user to look for. I would have searched for the text label instead of knowing from the docs that I was supposed to be looking for a button, and giving a user that information upfront can save them a lot of heartache.